internal/errors/resolver/resolver.go

```diff
@@ -56,11 +56,16 @@ func ExecuteTemplate(text string, data interface{}) (string, bool) {
 	if execErr != nil {
 		panic(fmt.Errorf("error executing template: %w", execErr))
 	}
-	return strings.TrimSpace(b.String()), true
+	return strings.TrimSpace(b.String())
+}
+
+type ResolvedErr struct {
```

Comment:
nit: rename it to some like `ResolvedResult` because `xxxErr` indicates it's an error.